bigbluebutton-html5/private/config/settings.yml

```diff
@@ -927,13 +927,9 @@ public:
       - 0.1
       - 0.2
     rtt:
-      - 500
-      - 1000
-      - 2000
-    level:
-      - warning
-      - danger
-      - critical
+      500: warning
+      1000: danger
+      2000: critical
```

Comment:
This pattern is confusing for who will change the settings.
I suggest to invert it to have the prop name in the left, like this:
```
warning: 500
danger: 1000
critical: 2000
```